apps/src/util/i18nStringTracker.js

```diff
@@ -0,0 +1,40 @@
+import experiments from '@cdo/apps/util/experiments';
+
+// A map of "string source" -> "list of strings used"
+// For example: window.stringUsageSources['maze'] = ['level_1.instruction.header']
+// The data recorded in this will periodically be uploaded.
+if (!window.stringUsageSources) {
```

Comment:
what's the motivation behind attaching this to the global `window` object?

Reply:
I just wanted somewhere to buffer them and have evidence that it's intercepting the string usage and logging it. 
In a follow-up PR, where I actually use the buffer where the string usage is logged, I will re-evaluate what the best way to implement this buffer is. I will follow-up with you offline to see if you have any recommendations.